pkgs/tools/misc/barman/default.nix

```diff
@@ -36,6 +37,9 @@ python3Packages.buildPythonApplication rec {
   disabledTests = [
     # Assertion error
     "test_help_output"
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
```suggestion
  ] ++ lib.optionals python3Packages.stdenv.isDarwin [
```

Reply:
Having not found `python3Packages.stdenv` in nixpkgs, I doubt it will work...

Reply:
`python3Packages.stdenv` evals to a stdenv for me.